islandora.module

```diff
@@ -624,6 +631,10 @@ function islandora_permission() {
       'title' => t('Replace datastreams'),
       'description' => t('Add new datastream content as latest version.'),
     ),
+    'manage orphaned objects' => array(
```

Comment:
We generally find it good practice to have permissions as constants.  It reduces future error.

Reply:
Not exactly sure what you mean there; I've modified to match what others are doing and hopefully that will be right. (Pushing the commit soon.)

Reply:
Using constants for important strings reduces transcription errors. When an undefined constant is used it is made obvious to developers, however using an incorrect literal string is not.

Reply:
^^^ permissions in particular can have quite extreme consequences in cases of transcription errors.